examples/demo-simple-stf/src/lib.rs

```diff
@@ -49,13 +56,14 @@ impl<Vm: Zkvm, Cond: ValidityCondition, B: BlobReaderTrait> StateTransitionFunct
 
     // Perform one-time initialization for the genesis block.
     fn init_chain(&mut self, _params: Self::InitialState) {
-        // Do nothing
+        // Do nothing and return an empty state root
     }
 
-    fn apply_slot<'a, I, Data>(
+    fn apply_slot<'a, I>(
         &mut self,
         _witness: Self::Witness,
-        _slot_data: &Data,
+        _slot_header: &Da::BlockHeader,
```

Comment:
Nit: I think we should be consistent in using `Da` vs `DA` capitalizing. I believe it is better to stick to single one.

Reply:
And Now I look at it and naming convention, we should probably convert all `DA` to `Da`